lib/rucio/core/nongrid_trace.py

```diff
@@ -31,18 +31,34 @@
 from rucio.common.config import config_get, config_get_int
 from rucio.core.monitor import record_counter
 
-ERRLOG = logging.getLogger('errlog')
-ERRLOG.setLevel(logging.ERROR)
+CONFIG_COMMON_LOGLEVEL = getattr(logging, config_get('common', 'loglevel', raise_exception=False, default='DEBUG').upper())
+CONFIG_COMMON_LOGFORMAT = config_get('common', 'logformat', raise_exception=False, default='%(asctime)s\t%(process)d\t%(levelname)s\t%(message)s')
```

Comment:
why do we need this here again? Can't we re-use this from `lib/rucio/common/logging.py`?

Reply:
The two trace modules are a bit special. I want a specific logger that write only the raw traces two a rotating log file as a buffer. The problem is, if I only setup the rotating logger, then these logs will also be duplicated to the root logger. That's why I added this hack to remove the handlers from the root logger and reconfigure it here, therefore I need these confit values again.